openjdk/ProblemList_openjdk13-openj9.txt

```diff
@@ -51,47 +51,44 @@ java/lang/StackWalker/VerifyStackTrace.java	https://github.com/AdoptOpenJDK/open
+java/lang/Throwable/SuppressedExceptions.java	https://github.com/eclipse/openj9/issues/6692	generic-all
+java/lang/annotation/AnnotationsInheritanceOrderRedefinitionTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1297	generic-all
+java/lang/annotation/loaderLeak/Main.java	https://github.com/eclipse/openj9/issues/6701	generic-all
 java/lang/invoke/AccessControlTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1285	generic-all
 java/lang/invoke/ArrayConstructorTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1288	generic-all
 java/lang/invoke/CallerSensitiveAccess.java	https://github.com/eclipse/openj9/issues/6768	generic-all
 java/lang/invoke/ClassSpecializerTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1297	generic-all
-java/lang/invoke/CustomizedLambdaFormTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
-java/lang/invoke/DefineClassTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
-java/lang/invoke/DropLookupModeTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
-java/lang/invoke/ExplicitCastArgumentsTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
-java/lang/invoke/FindAccessTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
-java/lang/invoke/InvokeMethodHandleWithBadArgument.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
-java/lang/invoke/InvokeWithArgumentsTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
-java/lang/invoke/LFCaching/LFMultiThreadCachingTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
-java/lang/invoke/LFCaching/LFSingleThreadCachingTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
-java/lang/invoke/LambdaFormTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/585	generic-all
+java/lang/invoke/DefineClassTest.java		https://github.com/eclipse/openj9/issues/6800	generic-all
+java/lang/invoke/DropLookupModeTest.java	https://github.com/eclipse/openj9/issues/6802	generic-all
+java/lang/invoke/ExplicitCastArgumentsTest.java		https://github.com/eclipse/openj9/issues/6856	generic-all
+java/lang/invoke/FindAccessTest.java		https://github.com/eclipse/openj9/issues/6868	generic-all
+java/lang/invoke/InvokeMethodHandleWithBadArgument.java		https://github.com/eclipse/openj9/issues/6943	generic-all
+java/lang/invoke/InvokeWithArgumentsTest.java	https://github.com/eclipse/openj9/issues/6964	generic-all
+java/lang/invoke/LFCaching/LFMultiThreadCachingTest.java	https://github.com/eclipse/openj9/issues/6963	generic-all
+java/lang/invoke/LFCaching/LFSingleThreadCachingTest.java	https://github.com/eclipse/openj9/issues/6963	generic-all
+java/lang/invoke/LambdaFormTest.java	https://github.com/AdoptOpenJDK/openjdk-tests/issues/1326	generic-all
```

Comment:
Needs updating to https://github.com/eclipse/openj9/issues/7055

Reply:
Actually a bunch of these have been permanently excluded so will update that